docs/contributing.rst

```diff
@@ -19,6 +19,11 @@ In particular, we need help in the following areas:
   `up-for-grabs <https://github.com/ethereum/solidity/issues?q=is%3Aopen+is%3Aissue+label%3Aup-for-grabs>`_ which are
   meant as introductory issues for external contributors.
 
+Please note that this project is released with a `Contributor Code of Conduct
+  <https://raw.githubusercontent.com/ethereum/solidity/develop/CODE_OF_CONDUCT.md>`_.
```

Comment:
I think you cannot indent here, at least it does not compile.

Reply:
Should be fixed.